test/common/upstream/upstream_impl_test.cc

```diff
@@ -723,13 +723,17 @@ TEST(ClusterMetadataTest, Metadata) {
     lb_policy: ROUND_ROBIN
     hosts: [{ socket_address: { address: foo.bar.com, port_value: 443 }}]
     metadata: { filter_metadata: { com.bar.foo: { baz: test_value } } }
+    common_lb_config:
+      healthy_panic_threshold:
+        value: 0.3
```

Comment:
Is this allowed to be less than one?  Isn't our validation supposed to catch this?  Or is this where we need the Init() hook in proto to get consistent validation?

Reply:
It's 0.0-1.0, right?

Reply:
healthy_panic_threshold ->    core.Percent healthy_panic_threshold = 1;
message Percent {
  double value = 1 [(validate.rules).double = {gte: 0, lte: 100}];
}

These are not the percents you're looking for :-P

Reply:
Sorry I mistyped above. I really do realize it's 0.0-100.0 now. :)

Are we good here though? 0.3 is valid?

Reply:
Ah, yeah, indeed.   And here I forgot it was a duoble this time :-)